src/Menu/ExpandMacros.cpp

```diff
@@ -153,12 +153,12 @@ ExpandTaskMacros(const TCHAR *name,
       case TaskAdvance::TURN_DISARMED:
 
         if (has_optional_starts && !common_stats.active_has_previous) {
-          return _("Next Startpoint");
+          return _("Startpoint Next");
         } else {
           invalid |= !common_stats.active_has_previous;
           return previous_is_start
-            ? _("Start Turnpoint")
-            : _("Previous Turnpoint");
+            ? _("Turnpoint Start")
+            : _("Turnpoint Previous");
```

Comment:
Hm, what's the motivation behind this? "Start turnpoint" sounds better than "Turnpoint Start" to me (I'm not a native English spearker though).

Reply:
Thx for reviewing, and starting a discussion. 

The idea was to have a bigger to smaller kind of ordering. So first ```context``` and then an ```action```.
All other labels on the buttons are already in that format:
Zoom +, or Task Abort 
So the idea would be to look for Turnpoint first, and then the aciton. Its not necessarily meant to be "written" english.

Reply:
I see, thanks for explanation!